src/components/Faq/styles.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+// import { bold } from 'ansi-colors';
+// why cant it be style.title? shouldnt it create a tag <title> and I shiuld be able to style it
+
+var FaqStyleContainer = styled.div`
+  width: 80%; 
+  max-width: 600%;
+  margin: 50px auto;
+`;
+
+var FaqStyleTitle = styled.div`
+&:hover {
+  background-color: #ddd;
+}
+  font-weight: bold;
+  font-size: 18px;
+  background-color: whitesmoke;
```

Comment:
Remember the variables.js folder I told you about? I'd suggest importing it into this file and using one of (or adding one) the variables we already have defined. 
`
import { variables } from '../../style'

background-color: ${variables.lightGrey};
`